website/redirects.next.js

```diff
@@ -1182,6 +1182,36 @@ module.exports = [
     destination: '/use-cases/network-infrastructure-automation',
     permanent: true,
   },
+  {
+    source: '/(/downloads?[-_]tools)',
+    destination: '/docs/download-tools',
+    permanent: true,
+  },
+  {
+    source: '/docs/k8s/ambassador',
+    destination: '/docs/k8s/connect/ambassador',
+    permanent: true,
+  },
+  {
+    source: '/docs/agent/config-entries/:path',
+    destination: '/docs/connect/config-entries/:path*',
+    permanent: true,
+  },
```

Comment:
We do still have https://www.consul.io/docs/agent/config-entries and https://www.consul.io/docs/connect/config-entries. Can you double check that these URLs still work, including with a trailing `/`? I'm not sure how `:path*` matches.

Reply:
yup, both resolve on the deploy preview without redirect and with trailing slash:
1. https://consul-6t5xh1g9i.vercel.app/docs/agent/config-entries
2. https://consul-6t5xh1g9i.vercel.app/docs/connect/config-entries